src/router/index.js

```diff
@@ -2,35 +2,92 @@ import Vue from "vue";
 import VueRouter from "vue-router";
 import store from '@/store'
 
+let Home;
+let NotFound;
+let Record;
+let Records
+let NewRecord;
+let Editor;
+let Login;
+let Signup;
+let ConfirmAccount;
+let ResendConfirmation;
+let User;
+let Curator;
+let RequestNewPassword;
+let ResetPassword;
+let EditProfile;
+let OauthLogin;
+let LoginFailure;
+let Stat;
+let Community;
+let Stakeholders;
+let Timeline;
+let License;
+let Terms;
+let Educational;
+let Privacy;
```

Comment:
You can declare on the let without commas without having to reuse the LET keyword. Works on single and multiple lines.
```suggestion
let License, Terms, Educational, Privacy, ... ;
```